core/src/main/java/org/apache/accumulo/core/clientImpl/TableOperationsImpl.java

```diff
@@ -525,7 +526,7 @@ public void addSplits(String tableName, SortedSet<Text> splits)
             }, tableName);
           } catch (TableExistsException | NamespaceExistsException | NamespaceNotFoundException
               | AccumuloSecurityException | TableNotFoundException | AccumuloException e) {
-            throw new RuntimeException(e);
+            throw new CompletionException(e);
```

Comment:
I don't know that changing this from a `RuntimeException` to a `CompletionException` will make any difference for the user. It doesn't really tell them anything beyond what `RuntimeException` signifies.

Reply:
My explanation for this PR was awful, let me fix that.  The following code used to run when adding splits before recent changes in elasticity from Future to CompletableFuture.  That code was dropped and this PR is restoring it because some ITs were breaking.

https://github.com/apache/accumulo/blob/e68f9dd2ef85677f52b922c4e34ef07bd1e13e15/core/src/main/java/org/apache/accumulo/core/clientImpl/TableOperationsImpl.java#L497

The CompletionException was something I learned about when trying to fix this and using it in the background thread makes the code running in the foreground thread shorter.  

I added a comment in ef6fb08 to call out why the exception type is important.